src/main/java/fr/esgi/al5_2/Tayarim/TayarimApplication.java

```diff
@@ -16,92 +16,90 @@
-    //     return args -> {
-    //         try (Connection conn = dataSource.getConnection()) {
-    //             // Récupère les métadonnées de la base de données
-    //             var metadata = conn.getMetaData();
-    //             // Récupère la liste des tables
-    //             try (ResultSet rs = metadata.getTables(null, null, "%", new String[] { "TABLE" })) {
-    //                 System.out.println("Liste des tables :");
-    //                 while (rs.next()) {
-    //                     // Affiche le nom de chaque table
-    //                     System.out.println("- " + rs.getString("TABLE_NAME"));
-    //                 }
-    //             }
-    //         } catch (SQLException e) {
-    //             e.printStackTrace();
-    //         }
-    //     };
-	//}
-
-	// @Bean
-	// public CommandLineRunner showTableSchemas(DataSource dataSource) {
-	// 	return args -> {
-	// 		try (Connection conn = dataSource.getConnection()) {
-	// 			DatabaseMetaData metadata = conn.getMetaData();
-
-	// 			// Récupère la liste des tables
-	// 			try (ResultSet tables = metadata.getTables(null, null, "%", new String[]{"TABLE"})) {
-	// 				while (tables.next()) {
-	// 					String tableName = tables.getString("TABLE_NAME");
-	// 					System.out.println("Table: " + tableName);
-
-	// 					// Récupérer les détails des colonnes pour chaque table
-	// 					try (ResultSet columns = metadata.getColumns(null, null, tableName, "%")) {
-	// 						while (columns.next()) {
-	// 							String columnName = columns.getString("COLUMN_NAME");
-	// 							String columnType = columns.getString("TYPE_NAME");
-	// 							int columnSize = columns.getInt("COLUMN_SIZE");
-
-	// 							System.out.println("\tColumn Name: " + columnName);
-	// 							System.out.println("\t\tType: " + columnType);
-	// 							System.out.println("\t\tSize: " + columnSize);
-	// 						}
-	// 					}
-	// 					System.out.println(); // Saut de ligne entre les tables pour une meilleure lisibilité
-	// 				}
-	// 			}
-	// 		} catch (SQLException e) {
-	// 			e.printStackTrace();
-	// 		}
-	// 	};
-
-    // }
\ No newline at end of file
+// public CommandLineRunner showTableSchemas(DataSource dataSource) {
+// 	return args -> {
+// 		try (Connection conn = dataSource.getConnection()) {
+// 			DatabaseMetaData metadata = conn.getMetaData();
+
+// 			// Récupère la liste des tables
+// 			try (ResultSet tables = metadata.getTables(null, null, "%", new String[]{"TABLE"})) {
+// 				while (tables.next()) {
+// 					String tableName = tables.getString("TABLE_NAME");
+// 					System.out.println("Table: " + tableName);
+
+// 					// Récupérer les détails des colonnes pour chaque table
+// 					try (ResultSet columns = metadata.getColumns(null, null, tableName, "%")) {
+// 						while (columns.next()) {
+// 							String columnName = columns.getString("COLUMN_NAME");
+// 							String columnType = columns.getString("TYPE_NAME");
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.sizes.LineLengthCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line is longer than 100 characters (found 107).